org-code-javabuilder/playground/src/main/java/org/code/playground/ClickableItem.java

```diff
@@ -0,0 +1,20 @@
+package org.code.playground;
+
+import java.io.FileNotFoundException;
+
+public abstract class ClickableItem extends Item {
```

Comment:
Discussed on Slack a bit already but just wanted to add a few thoughts - we decided that `ClickableItem` would not extend `Image`/depend on the `media` package to avoid confusion and that definitely makes sense. My only concern is that if we do decide later on to support more complicated image manipulation here (e.g. manipulating individual pixels/colors) then it will be a little more effort to add that capability back in as we'll have to bring back `Image` in some way. So as long as we're only ever going to allow users to display an image and only manipulate its position and width/height, then this will work, but just wanted to call this out.

Reply:
Similarly, I had thought we are planning to allow students to draw shapes - is this something we decided against?

Reply:
Right now the only capabilities we know are needed are showing images at a specific position/size, but I do see a world where we would want to add pixel manipulation. I think that this could be pretty straightforward to add by creating new constructors for `ClickableItem` and `Item` that takes an `Image` object. If we wanted to add the ability to manipulate the image of an already showing `Item`/`ClickableItem`, e.g. `updateImage()` or `setImage(Image img)`/`setImage(String filename)`. But actually implementing those methods would be more involved.

As for shape drawing, it was a nice-to-have so I omitted it. We can add APIs in the future for that as well.

Reply:
@sanchitmalhotra126, how would supporting or not supporting shape drawing affect how we design the API?

Reply:
It will depend on implementation but if we are able to reuse Theater code using the more backend approach, then it won't be too much work to support drawing shapes, especially if we already have to support drawing text. We would be able to reuse any drawing functionality that exists in Theater already (text, lines, shapes).

Reply:
👍